### PR TITLE
Support logic json format in TensileLite

### PR DESCRIPTION
**Pull request by KKyang**

### CODE REVIEW

**Review by hcman2 — APPROVED** [review]

File: tensilelite/Tensile/LibraryIO.py [hunk 1]
```diff
@@ -30,6 +30,22 @@
...
+except ImportError:
+    try:
+        import ujson as json
+        printWarning("orjson not installed. Fallback to ujson.")
+    except ImportError:
+        try:
+            import simplejson as json
+            printWarning("orjson, ujson not installed. Fallback to simplejson.")
+        except ImportError:
+            import json
```
Comment (AndySu12) [inline tensilelite/Tensile/LibraryIO.py:47]: Why not just using "json" but considering all variants? Apart from encoding, is there any further concern?

**Review by AndySu12 — DISMISSED** [review]

File: tensilelite/Tensile/LibraryIO.py (hunk 1 above)
Comment (KKyang) [reply tensilelite/Tensile/LibraryIO.py:47]: There are performance gaps between these packages.

Comment (AndySu12) [reply tensilelite/Tensile/LibraryIO.py:47]: OK got it.

File: tensilelite/Tensile/LibraryIO.py [hunk 2]
```diff
@@ -143,16 +166,30 @@ def writeSolutions(filename, problemSizes, biasTypeArgs, activationArgs, solutio
...
+        printExit("Unrecognized read format {}".format(extension))
+
 def readYAML(filename):
     """Reads and returns YAML data from file."""
     with open(filename, "r") as f:
         data = yaml.load(f, yamlLoader)
     return data
 
+def readJson(filename):
+    """Reads and returns YAML data from file."""
```
Comment (Serge45) [inline tensilelite/Tensile/LibraryIO.py]: nitpick: JSON.

**Review by Serge45 — APPROVED** [review]
LGTM.

File: tensilelite/Tensile/LibraryIO.py (hunk 2 above)
Comment (KKyang) [reply tensilelite/Tensile/LibraryIO.py]: :(

**Pull request merged**